docs/portal/dashboard.md

```diff
@@ -1,9 +1,26 @@
-# Understanding the Dashboard
+# Dashboard
 
-The Dashboard appears when you first log in to the {{title}} portal.
+HDT Dashboard give you information about the traffic of your HDT transports.
 
-Its charts provide a snapshot of your account's traffic. Clicking **View Full Report** in the upper right corner of the chart displays the chart on the [Reports page](</docs/portal/reports.md>), where you can define report parameters, and then view the results on the selected chart.
+If you are a reseller with child accounts, you can select the accounts that the dashboards cover (the parent account only, the child account only, or both the parent and child accounts).
 
-![null](</docs/resources/images/dashboard.png>)
+## Understanding the Dashboard
+
+The Dashboard appears when you first log into the {{title}} portal. The charts provide a snapshot of your account's traffic, covering "Traffic Volume", "Traffic Bandwidth" and "Requests".
+
+If you are a paid customer of **Virtual Private Line** usage, you can also switch between “Total” and “Virtual Private Line” dashboard by clicking the selection button at the upper right corner.
+
+![null](</docs/resources/images/dashboard/selectTotalOrVpl.png>)
+
+Holding your cursor over the data points in the report, then you can review the detailed information in the tooltips. 
+
+![null](</docs/resources/images/dashboard/dashboard.png>)
+
+The default time zone in the tooltips is from the local browser time. You can also overwrite it by changing the time zone in "My Info".
+
+![null](</docs/resources/images/dashboard/myInfo.png>)
+
+Clicking **View Full Report** in the upper right corner of the chart will display the chart on the [Reports page](</docs/portal/reports.md>), where you can define report parameters, and then view the results on the selected chart.
```

Comment:
How about: "Clicking **View Full Report** in the top-right corner of the chart  displays a chart on the [Reports page](</docs/portal/reports.md>), where you can define report parameters, and then view the results on the selected chart."

Reply:
OK, will update it.